synapse/lib/stormtypes.py

```diff
@@ -6595,6 +6636,10 @@ def _formatTimestamp(ts):
 
     async def _methCronJobPprint(self):
         user = self.valu.get('username')
+        view = self.valu.get('view')
+        if not view:
+            view = 'default'
```

Comment:
Shouln't this be the cortex default view iden?

Reply:
It is now :smile: